app/errors.py

```diff
@@ -0,0 +1,14 @@
+class NotWearingMaskError(BaseException):
```

Comment:
Use `Exception` for user-defined errors
https://docs.python.org/3/library/exceptions.html#BaseException